packages/venia-ui/lib/components/Navigation/navHeader.js

```diff
@@ -21,34 +18,36 @@ const titles = {
 };
 
 const NavHeader = props => {
-    const { isTopLevel, onBack, onClose, view } = props;
```

Comment:
You got rid of this prop but its still being passed [in the parent](https://github.com/magento/pwa-studio/pull/2550/files#diff-850d347f8af9be997d875083aa44d202R60).

Reply:
Thanks! https://github.com/magento/pwa-studio/pull/2550/commits/119294e6c917a4db74db1f74a08e4a108747988d.